assistant/execution/executor.py

```
@@ -119,7 +119,8 @@ def _prepare_arguments(self, function: Callable, arguments_json: str) -> Dict[st
             raise ToolExecutionError(
                 message=f"Failed to process arguments: {str(e)}",
                 tool_name=function.__name__,
-                tool_args=arguments_json
+                tool_args=arguments_json,
+                details={"function_name": function.__name__, "arguments_json": arguments_json, "tool_call_id": context.tool_call_id}
```

Ruff (0.8.2)

Undefined name `context`

(F821)